packages/radio-group/src/vaadin-radio-group.js

```diff
@@ -73,510 +85,380 @@ class RadioGroupElement extends ThemableMixin(DirMixin(PolymerElement)) {
           display: none !important;
         }
 
-        .vaadin-group-field-container {
+        [part='container'] {
```

Comment:
I'm not sure if we planned to introduce the new `container` part. The new part was brought from the prototype.

Reply:
Discussed and decided to keep the part.